assets/javascript/app.js

```diff
@@ -105,12 +111,15 @@ $(document).on('ready', function () {
         if (input.match(/^[\w\-\s]+$/)) {
             showLoad();
             animate();
-            displayInputBeer();
+            displayInputBeer(input);
             setTimeout(function () {
                 fetchBeers();
-            }, 5000);
+            }, 3000);
         } else {
             showInvalidCharacterError();
+            setTimeout(function () {
+                hideInvalidCharacterError();
+            }, 10000);
```

Comment:
@mdvb1001 do things following this continue to run or does everything stop for 10 seconds?